### PR TITLE
update follow button in series-list and project's intro

### PR DESCRIPTION
**Pull request by kwhsiung**

### CODE REVIEW

**Comment by kwhsiung** [comment]
[專題頁增加追蹤功能（增加追蹤按鈕）](https://trello.com/c/fQaxB4Km/314-%E5%B0%88%E9%A1%8C%E9%A0%81%E5%A2%9E%E5%8A%A0%E8%BF%BD%E8%B9%A4%E5%8A%9F%E8%83%BD)

File: src/components/AppArticleNav.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <CommentCount class="comment-icon__count" :commentAmount="commentCount" :assetUrl="assetUrl" :postId="postId" :type="'publicPostsHot'"></CommentCount>
       </span>
       <!-- <img class="follow-icon" :src="isFollow ? '/public/icons/star-blue.png' : '/public/icons/star-line-blue.png'" alt="follow" @click="toogleFollow"> -->
-      <span v-if="isLoggedIn" class="follow-icon" @click="toogleFollow($event)">
+      <span v-if="isLoggedIn && showFollow" class="follow-icon" @click="toogleFollow($event)">
         <img class="follow-icon__thumbnail" :src="isFollow ? '/public/icons/star-blue.png' : '/public/icons/star-line-blue.png'" alt="follow">
```
Comment (hcchien) [inline src/components/AppArticleNav.vue]: so the button won't display if the user didn't log in?

Comment (kwhsiung) [reply src/components/AppArticleNav.vue]: Yes

Comment (hcchien) [reply src/components/AppArticleNav.vue]: I think we should redirect to the log in page if the users didn't log in, and always display the button.

**Pull request merged**